samples/iolib/src/main/cpp/player/SampleBuffer.cpp

```diff
@@ -18,24 +18,22 @@
 
 #include "io/wav/WavStreamReader.h"
 
-using namespace parselib;
-
 namespace iolib {
 
-void SampleBuffer::loadSampleData(WavStreamReader* reader) {
+void SampleBuffer::loadSampleData(parselib::WavStreamReader* reader) {
     mAudioProperties.channelCount = reader->getNumChannels();
     mAudioProperties.sampleRate = reader->getSampleRate();
 
     reader->positionToAudio();
 
     mNumSampleFrames = reader->getNumSampleFrames() * reader->getNumChannels();
```

Comment:
This should be mNumSamples. It is not equal to numFrames except when mono.

Reply:
The whole thing needs to be generalized for arbitrary channels. Still, done.